driver/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+################################################################################
+# Mock host model drivers
+
+set( DRIVER_F90_SRC driver/compute_aero_optics.F90
```

Comment:
Any reason not to use `add_executable` here? We can always have them do a `make install` that splats the driver into a set location.

Reply:
I'm not sure I follow. Right now the drivers ends up in the build folder. Where were you proposing that they end up?